banners/migrations/0002_auto_20190411_1521.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 2.0.9 on 2019-04-11 15:21
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
Do you plan to squash these two migrations when design of the `Banner` model is finished?

Reply:
aye